flink-staging/flink-table/pom.xml

```diff
@@ -59,6 +59,12 @@ under the License.
 		</dependency>
 
 		<dependency>
+			<groupId>org.apache.flink</groupId>
+			<artifactId>flink-hcatalog</artifactId>
+			<version>${project.version}</version>
+		</dependency>
+
+		<dependency>
```

Comment:
What do you mean with "put into another package"? Do you mean putting Table API classes in the `flink-hcatalog` package? I think thats a general decision we have to make. Or do we want additional Maven Table API I/O format modules? For Parquet, HCat. etc...